content/en/docs/concepts/_index.md

```diff
@@ -17,7 +17,7 @@ The Concepts section helps you learn about the parts of the Kubernetes system an
 
 To work with Kubernetes, you use *Kubernetes API objects* to describe your cluster's *desired state*: what applications or other workloads you want to run, what container images they use, the number of replicas, what network and disk resources you want to make available, and more. You set your desired state by creating objects using the Kubernetes API, typically via the command-line interface, `kubectl`. You can also use the Kubernetes API directly to interact with the cluster and set or modify your desired state.
 
-Once you've set your desired state, the *Kubernetes Control Plane* works to make the cluster's current state match the desired state. To do so, Kubernetes performs a variety of tasks automatically--such as starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster: 
+Once you've set your desired state, the *Kubernetes Control Plane* using the Pod Lifecycle Event Generator (PLEG) works to make the cluster's current state matches the desired state. To do so, Kubernetes performs a variety of tasks automatically--such as starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster: 
```

Comment:
Hello @thecrudge. Thanks for the PR.
- Check the commit message, see the note about referencing issues in commit messages:
https://kubernetes.io/docs/contribute/intermediate/#clone-the-repository
- I am having trouble parsing the changes.

Reply:
Try now, I've amended my commit message